src/Components/MainPage/PositionDetailed/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import useQuery from 'Hooks/useQuery';
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { getPositionById } from 'redux/positions/thunks';
+import { cleanSelectedItem } from 'redux/interviews/actions';
+import { useHistory } from 'react-router-dom';
+
+function PositionDetailed() {
+  const history = useHistory();
+  const query = useQuery();
+  const dispatch = useDispatch();
+  const selectedItem = useSelector((store) => store.positions.selectedItem);
+
+  const positionId = query.get('_id');
+  useEffect(() => {
+    if (positionId) {
+      dispatch(getPositionById(positionId));
+    }
+  }, []);
+
+  useEffect(() => {
+    return () => {
+      dispatch(cleanSelectedItem());
+    };
+  }, []);
+
+  return (
+    <div>
+      <div>
+        <h3>{selectedItem.professionalProfile?.name}</h3>
+        <p>{selectedItem.client?.name}</p>
+        <p>{selectedItem.createdAt}</p>
```

Comment:
This field should only show the date, it doesn't matter what time the position was opened